roles/oneagent/tests/component/scripts/command/windows/windows_command_executor.py

```diff
@@ -0,0 +1,14 @@
+import winrm
+
+from util.test_data_types import CommandResult
+
+
+class WindowsCommandExecutor:
+    def __init__(self, user: str, password: str):
+        self.user = user
+        self.password = password
+
+    def execute(self, address: str, command: str, *args: str) -> CommandResult:
+        session = winrm.Session(address, auth=(self.user, self.password))
```

Comment:
Here, the issue with session management is even better visible than in ssh case, as pywinrm sessions are designed to be maintained for some longer period of time.